ntc_templates/templates/alcatel_aos_show_port-security.textfsm

```diff
@@ -0,0 +1,24 @@
+Value PORT ((\d+\/?)+)
+Value OPERATION_MODE (\S+)
+Value ADMIN_STATE (\S+)
+Value MAX_MAC_BRIDGED (\d+)
+Value TRAP_THRESHOLD (\S+)
+Value MAX_MAC_FILTERED (\d+)
+Value VIOLATION_TYPE (\S+)
+Value VIOLATION_MAC (\S+)
+Value PKT_RELAY (\S+)
+
+
+Start
+  ^\s*Port:\s*${PORT}\s*$$
```

Comment:
```suggestion
  ^Port: -> Continue.Record
  ^\s*Port:\s*${PORT}\s*$$
```
Your current test is missing the first result, this will capture both interfaces